charts/newrelic-infrastructure/templates/kubelet/daemonset.yaml

```diff
@@ -99,8 +99,8 @@ spec:
           resources: {{ toYaml . | nindent 12 }}
           {{- end }}
         - name: agent
-          image: "{{ .Values.images.agent.registry }}/{{ .Values.images.agent.repository }}:{{ .Values.images.agent.tag }}"
-          command: [ "/sbin/tini", "--", "/usr/bin/newrelic-infra" ]
```

Comment:
why removing tini?

Reply:
CAOS has [`tini` in the entrypoint and `/usr/bin/newrelic-infra-service` in the command](https://github.com/newrelic/infrastructure-agent/blob/master/build/container/Dockerfile#L44-L45)

I don't want to override `tini` so CAOS can change PID 1 freely but only change the command from `newrelic-infra-service` to `newrelic-infra`.

```diff
- command: [ "/sbin/tini", "--", "/usr/bin/newrelic-infra" ]
+ args: [ "newrelic-infra" ]
```

Do you see any issues?